libs/blocks/section-metadata/section-metadata.css

```diff
@@ -204,6 +204,11 @@
 }
 
 .section.sticky-bottom.promo-sticky-section {
+  background: none;
+  z-index: 4;
```

Comment:
Could we have variables around this?

E.g. 
`--base-z-index: 0`
`--promobar-z-index: calc(var(--base-z-index) + 1)` or whatever you base the value on?

Reply:
Technically, it can be done, but I’m uncertain whether it’s the best approach. What are your thoughts?

Here’s the current setup:

```
.section picture.section-background {
  display: block;
  position: absolute;
  inset: 0;
  z-index: -1;
}
.section.sticky-top {
  position: sticky;
  top: 57px;
  z-index: 1;
  background-color: var(--color-white);
}
.section.sticky-bottom {
  position: sticky;
  bottom: 0;
  z-index: 3;
  background-color: var(--color-white);
}
.section.sticky-bottom.promo-sticky-section {
  background: none;
  z-index: 4;
}
.section.sticky-bottom.promo-sticky-section.popup {
  background: none;
  z-index: 40000;
}

```
We could define a variable for the z-index: 3 in .section.sticky-bottom, and then increment it to 4 for .section.sticky-bottom.promo-sticky-section.

Should we proceed with this? Or would you recommend setting variables for all the z-index values in the file?

Reply:
I wonder if we should just ad this to `styles.css` and can then have the obvious variables and the z-index: +1, to have a good strategy.

Or potentially more general levels
- base-z-index: 0
- modals: 1
- promos: 2
- above-all: 9999? 

... @overmyheadandbody any better ideas than the two?

Reply:
I also think we should establish a clear sequence for what elements are layered on top of each other and keep all the z-index values in one place. This way, the next time someone adds an element, they'll have a clear idea of the existing layer order.